airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx

```diff
@@ -266,6 +268,11 @@ export const AuthenticationProvider: React.FC<React.PropsWithChildren<unknown>>
         await authService.sendEmailVerifiedLink();
 
         if (auth.currentUser) {
+          // exp-speedy-connection
+          localStorage.setItem(
```

Comment:
Set the timestamp when the user is signed up

Reply:
This only sets this if users sign up via username/password but not via OAuth. Is this intended behavior?

Reply:
Thanks! I forgot about OAuth